[2025-05-25T04:08:26.756+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-25T04:08:26.818+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_csv_pipeline_v2.load_data manual__2025-05-25T04:07:00.877135+00:00 [queued]>
[2025-05-25T04:08:26.858+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_csv_pipeline_v2.load_data manual__2025-05-25T04:07:00.877135+00:00 [queued]>
[2025-05-25T04:08:26.901+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-05-25T04:08:26.945+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): load_data> on 2025-05-25 04:07:00.877135+00:00
[2025-05-25T04:08:26.970+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=499) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-25T04:08:26.986+0000] {standard_task_runner.py:72} INFO - Started process 501 to run task
[2025-05-25T04:08:26.982+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'etl_csv_pipeline_v2', 'load_data', 'manual__2025-05-25T04:07:00.877135+00:00', '--job-id', '211', '--raw', '--subdir', 'DAGS_FOLDER/etl_pipeline.py', '--cfg-path', '/tmp/tmpl6ysahy2']
[2025-05-25T04:08:26.988+0000] {standard_task_runner.py:105} INFO - Job 211: Subtask load_data
[2025-05-25T04:08:27.207+0000] {task_command.py:467} INFO - Running <TaskInstance: etl_csv_pipeline_v2.load_data manual__2025-05-25T04:07:00.877135+00:00 [running]> on host a9ca17474c33
[2025-05-25T04:08:28.536+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_csv_pipeline_v2' AIRFLOW_CTX_TASK_ID='load_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-25T04:07:00.877135+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-25T04:07:00.877135+00:00'
[2025-05-25T04:08:28.586+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-05-25T04:08:28.602+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-05-25T04:08:28.627+0000] {logging_mixin.py:190} INFO - Current task name:load_data state:running start_date:2025-05-25 04:08:26.820380+00:00
[2025-05-25T04:08:28.643+0000] {logging_mixin.py:190} INFO - Dag name:etl_csv_pipeline_v2 and current dag run status:running
[2025-05-25T04:08:28.655+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-05-25T04:08:28.911+0000] {load.py:30} INFO - âœ… Archivo final guardado en data/output/superstore_clean.csv
[2025-05-25T04:08:28.914+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-05-25T04:08:28.960+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-05-25T04:08:28.962+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=etl_csv_pipeline_v2, task_id=load_data, run_id=manual__2025-05-25T04:07:00.877135+00:00, execution_date=20250525T040700, start_date=20250525T040826, end_date=20250525T040828
[2025-05-25T04:08:29.038+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-05-25T04:08:29.039+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-05-25T04:08:29.044+0000] {logging_mixin.py:190} INFO - Dag name:etl_csv_pipeline_v2 queued_at:2025-05-25 04:07:01.282895+00:00
[2025-05-25T04:08:29.045+0000] {logging_mixin.py:190} INFO - Task hostname:a9ca17474c33 operator:PythonOperator
[2025-05-25T04:08:31.173+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-25T04:08:31.295+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-25T04:08:31.306+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-05-25T04:08:33.564+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
